### ISSUE TITLE
Execute console shell command without confirmation

### ISSUE THREAD

**Issue by Kerruba** [ranger/ranger#1410]
Hello,

mine is actually a quick question but I couldn't find an answer to this.

Is it possible to map a shell command to run immediately without confirmation?

I'm using Mac and I would like to map "pf" to open Preview on the selected files, but when I add the line to rc.conf, in the console I still have to type <Enter> if I want Preview to start

Here the map
```
map pf console shell open -a Preview %s
```

Do I have to add something to the end to execute the shell command directly?

**Comment by toonn** [comment]
Yes, `console` is just a command that opens up the console with whatever you pass it. Note that some commands still ask for confirmation, like `delete`. The console isn't actually about confirmation, it's about letting you edit the command, pass an argument or something. It's very useful for something like `scout`.
